Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -887,6 +892,9 @@ const TextInput = createReactClass({
         'TextInput children are not supported on Windows.'
     );
 
+    var tabIndex = this.props.tabIndex || 0;
```

Comment:
>var [](start = 4, length = 3)

Switch to `const` or `let`

Reply:
Changing lines I added. Everything else should be up to upstream.

Reply:
Changing lines I added. Everything else should be up to upstream.

---
In reply to: [156557760](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156557760) [](ancestors = 156557760)